scripts/create-clusters.sh

```diff
@@ -74,7 +75,7 @@ function configure-insecure-registry-and-reload() {
   local cmd_context="${1}" # context to run command e.g. sudo, docker exec
   local docker_pid="${2}"
   ${cmd_context} "$(insecure-registry-config-cmd)"
```

Comment:
If you consider my comment below, this line would have to be generic based on the 3rd parameter passed in.